packages/kit/src/loader/nuxt.ts

```diff
@@ -18,55 +19,87 @@ export interface LoadNuxtOptions extends LoadNuxtConfigOptions {
   config?: LoadNuxtConfigOptions['overrides']
 }
 
-export async function loadNuxt (opts: LoadNuxtOptions): Promise<Nuxt> {
-  // Backward compatibility
-  opts.cwd = opts.cwd || opts.rootDir
-  opts.overrides = opts.overrides || opts.config || {}
+export interface ImportNuxtPackageOptions {
+  cwd?: string
+  version?: 2 | 3
+}
 
-  // Apply dev as config override
-  opts.overrides.dev = !!opts.dev
+const nuxtPackages = {
+  2: ['nuxt-edge', 'nuxt'],
```

Comment:
Notice that `2` here is type of number while the `ImportNuxtPackageOptions["version"]` requires a number. It works with implicit type casting